server/src/interfaces/event.interface.ts

```diff
@@ -89,6 +88,13 @@ export type EventItem<T extends EmitEvent> = {
   server: boolean;
 };
 
+export enum BootstrapEventPriority {
```

Comment:
What is the direction of the priority level?

Reply:
Maybe we can add a comment for this info?

Reply:
If you think it's useful, I think it's implied by the existing comments and priorities in the enum already